ils_middleware/tasks/bluecore/storage.py

```diff
@@ -21,6 +21,19 @@ def store_bluecore_resources(**kwargs):
     Note: Virtualenv is needed because bluecore.models uses SQLAlchemy 2.+
     and Airflow uses a 1.x version of SQLAlchemy
     """
+    try:
+        """
+        Set CURRENT_USER_ID from DAG-provided user_uid so #add_version can write
+        versions.keycloak_user_id during ORM events triggered by inserts/updates.
+        """
+        from bluecore_models.models.version import CURRENT_USER_ID
+
+        uid = kwargs.get("user_uid")
+        CURRENT_USER_ID.set(uid)
+        print("Using CURRENT_USER_ID: ", uid)
```

Comment:
I think this print statement should be logged instead.

Reply:
Ok i updated this to use logging. I had to import logging in that method since it was running in a virtual environment, and would throw out errors otherwise.